cli/registry.go

```diff
@@ -0,0 +1,203 @@
+package run
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/docker/docker/api/types/network"
+	"github.com/docker/docker/client"
+	"github.com/rancher/k3s/pkg/agent/templates"
+	log "github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
+)
+
+// defaultRegistriesGlobalConfig is a global registries file that will be used in all the servers/workers
+const defaultRegistriesGlobalConfig = "$HOME/.k3d/registries.yaml"
```

Comment:
this would fail on windows, wouldn't it?
We use `"github.com/mitchellh/go-homedir"` to determine the homedir of the user cross-platform.